### PR TITLE
fix: separate models from recipes

### PR DESCRIPTION
**Pull request by lstocchi**

### CODE REVIEW

**Comment by feloy** [comment]
I amde a first test, but the description is not displayed in the Playground page, and I get an error with `model not found` when I try to send a query to the model (I'll have a look to the code to check where it fails)

File: packages/shared/StudioAPI.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ export abstract class StudioAPI {
   abstract getCategories(): Promise<Category[]>;
   abstract getRecipesByCategory(categoryId: string): Promise<Recipe[]>;
   abstract getRecipeById(recipeId: string): Promise<Recipe>;
-  abstract getModelById(modelId: string): Promise<ModelInfo>;
+  abstract getModelById(modelId: string): ModelInfo;
```
Comment (feloy) [inline packages/shared/StudioAPI.ts:14]: This needs to be returning a Promise, as the value will be returned by exchanging messags

**Comment by lstocchi** [comment]
@feloy thanks for pointing me to the problem. I should have fixed everything. Please give it a look 👍

**Review by feloy — APPROVED** [review]
works great, thanks

**Pull request merged**